build-logic/uber-plugins/src/main/kotlin/gradlebuild.distribution-module.gradle.kts

```diff
@@ -51,7 +51,7 @@ val apiStubElements = configurations.consumable("apiStubElements") {
 
 // FIXME Publishing API stubs for mixed Java/Kotlin subprojects don't work currently;
 //       we only publish the Kotlin stubs for some reason
-pluginManager.withPlugin("gradlebuild.jvm-library") {
+pluginManager.withPlugin("gradlebuild.java-library") {
```

Comment:
❓ Why does that fix the issue?

Reply:
As discussed: we then apply Java ABI extraction logic only for java-library. While jvm-library is applied also for Kotlin